File: Submissions/25b0970/Week 1/w1_q2.py
n=input()
a=len(n)
c=0
i=0
j=0
while i<a:
    b=1
    j=i+1
    while j<a:
 
        if n[i]==n[j]:
            b=b+1 
            c=max(c,b)
            j=j+1
            
        else:
            c=max(c,b)
            break
    i=j
    if a==1:
        c=1
print(c)

File: Submissions/24b0964/Week 1/w1_q2.cpp
#include <iostream> 
#include <string>
using namespace std;

int main(){
    string str;
    cin>>str;

    char prev = str[0];
    long long maxi = 0, count = 0;
    for(auto c : str){
        // cout<<c<<'\n';
        if(c==prev) count+=1;
        else{
            count = 1;
        }
        maxi = max(maxi, count);
        // cout<<maxi<<'\n';
        prev = c;
    }

    cout<<maxi<<'\n';
}
